### DISCUSSION TITLE
Adding a Ziegler-Biersack- Littmark potential

### DISCUSSION THREAD

**Discussion by fpriante** [mir-group/allegro#43] [General]
Hi, I saw in https://arxiv.org/abs/2304.10061 that you added a ZBL potential on top of Allegro to improve stability. I was wondering if you did it during training or if you just added it during production runs, and, in both cases, if you could give some suggestion on how to proceed about implementing it.

Thanks.

**Comment by Linux-cpp-lisp** [comment]
Hi @fpriante ,

Thanks for your interest in our work!

This is available when using the `develop` branch of `nequip` (0.6.*), see https://github.com/mir-group/nequip/blob/develop/configs/minimal_pair.yaml.

We use the ZBL potential in both training and inference.

**Reply by fpriante** [reply]
Thanks a lot! 
I see you also have a LJ there. In case you tried it, did it cause meaningful structural differences compared to simple Allegro in the large structures you simulate in the paper?

**Reply by Linux-cpp-lisp** [reply]
We did not use Lennard Jones in that effort;  do note that for LJ you have to provide the LJ parameters in the YAML input file and it is not as well tested as the ZBL implementation at this point.
